sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/ViewRegistry.java

```diff
@@ -55,17 +58,34 @@ public static ViewRegistryBuilder builder() {
    * @param descriptor description of the instrument.
    * @return The {@link View} for this instrument, or a default aggregation view.
    */
-  public View findView(InstrumentDescriptor descriptor) {
-    LinkedHashMap<Pattern, View> configPerType = configuration.get(descriptor.getType());
-    for (Map.Entry<Pattern, View> entry : configPerType.entrySet()) {
-      if (entry.getKey().matcher(descriptor.getName()).matches()) {
-        return entry.getValue();
+  public View findView(InstrumentDescriptor descriptor, InstrumentationLibraryInfo meter) {
+    for (RegisteredView entry : reverseRegistration) {
```

Comment:
I thought that Reilly's intent was to have the views applied with the first registered having priority. Any particular reason to reverse it here?

Reply:
See comment on the PR.  Mostly I kept the reversal behavior so I didn't break existing tests, otherwise need to shuffle them.  Combine that with the Specification not *clearly* denoting order.  I can flip it though if we think that's the way the spec is going.

Reply:
cool. no worries; I think this is something we can evolve, for sure.

Reply:
Personally, I think the way java had it leads to a much more flexible configuration model where "last added wins", if we combine this with a clear mechanism where configuration is loaded and in what order.